### PR TITLE
Mandiant v 1 1

### PR DESCRIPTION
**Pull request by adamlevymandiant**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
NA

## Description
New version that:

- Splits Feed and Enrichment commands into separate integrations
- Enhances the readable output for the `cve`, `mati-get-actor`, `mati-get-malware`, `mati-get-campaign` commands

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
@adamlevymandiant - Can you please sync the fork? It should resolve the [Setup Environment check failing](https://app.circleci.com/pipelines/github/demisto/content/379113/workflows/46eda44e-3d05-429b-83cf-e273ec14452a/jobs/751431) and the [pre-commit validation failure](https://github.com/demisto/content/actions/runs/10583280098/job/29354583437?pr=36047#step:8:831).


Also, can you please increase the code coverage for Feed Mandiant integration? Seems that we're [lower than the 70% threshold](https://github.com/demisto/content/actions/runs/10583280098/job/29354583437?pr=36047#step:8:1027).

**Comment by adamlevymandiant** [comment]
@kgal-pan - thank you for the guidance, I have synced my fork and pushed a commit increasing test coverage above 70%

**Comment by barryyosi-panw** [comment]
Hi @adamlevymandiant, since you were splitting the original integration into two separate ones, it still makes sense to use a shared client module for all the obvious reasons.
I'd suggest you to try creating a shared client module, and import it into the two integrations, let me know if you wish to try it and need assistance with that.

In addition, I see you've got some validations failing - try using `demisto-sdk format -i Packs/MandiantAdvantageThreatIntelligence/Integrations/Mandiant/Mandiant.yml` and 
`demisto-sdk format -i Packs/MandiantAdvantageThreatIntelligence/Integrations/FeedMandiant/FeedMandiant.yml`  

it should solve the formatting problems.

**Comment by adamlevymandiant** [comment]
@barryyosi-panw - thank you for the guidance, is there a pack that already uses a shared client module and custom docker image that I can use as reference?

**Comment by barryyosi-panw** [comment]
Surprisingly, it's actually a precedence, and we don't have a reference of such mechanism. 
Lets keep the current solution for now @adamlevymandiant, your PR will be reviewed shortly.

File: Packs/MandiantAdvantageThreatIntelligence/Integrations/FeedMandiant/FeedMandiant.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  hidden: true
+  name: feedIncremental
+  required: false
+  type: 8
+script:
+  commands:
+  - name: mandiant-get-indicators
+    arguments:
+    - description: The maximum number of indicators to fetch.
+      name: limit
```
Comment (kgal-pan) [inline Packs/MandiantAdvantageThreatIntelligence/Integrations/FeedMandiant/FeedMandiant.yml:127]: Please add a `description` key to the `mandiant-get-indicators` command to fix the [pre-commit validation failure](https://github.com/demisto/content/actions/runs/10697530257/job/29654917907?pr=36047#step:8:875).

```suggestion
  - name: mandiant-get-indicators
    arguments:
    - description: The maximum number of indicators to fetch.
      name: limit
    description: Fetch indicators.
```

Comment (kgal-pan) [reply Packs/MandiantAdvantageThreatIntelligence/Integrations/FeedMandiant/FeedMandiant.yml:127]: Pushed the change myself in https://github.com/demisto/content/pull/36047/commits/f3f3aba0911da1084edd1fe967857079d451cd6d.

**Comment by adamlevymandiant** [comment]
Thanks @kgal-pan - I see you have pushed some commits, do you need me to do anything at this stage?

**Comment by kgal-pan** [comment]
> Thanks @kgal-pan - I see you have pushed some commits, do you need me to do anything at this stage?

Sure thing @adamlevymandiant - I just pushed a commit to fix the graph/neo4j crashing in the pre-commit validation stage because of the missing `description` field for the command.

Now that this commit fixed the graph crash, it exposed some things that need to be fixed in the integration. Please review the listed errors starting from [here](https://github.com/demisto/content/actions/runs/10702193210/job/29669821546?pr=36047#step:8:834).

> Is it too late to change the email in `pack_metadata.json` to [mandiant-support@google.com](mailto:mandiant-support@google.com)?

No, feel free to change this.

**Comment by adamlevymandiant** [comment]
@kgal-pan - FYI that circleci checks are now passing. Please let me know if anything else is required from my side.

File: Packs/MandiantAdvantageThreatIntelligence/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#### Integrations
+##### Mandiant Enrich
+- New Integration providing generic enrichment commands.
+- Includes enhancements to the readable output for the `cve`, `mati-get-actor`, `mati-get-malware`, `mati-get-campaign` commands.
+##### Mandiant Feed
+- New Integration providing an indicator feed.
```
Comment (ShirleyDenkberg) [inline Packs/MandiantAdvantageThreatIntelligence/ReleaseNotes/1_1_0.md:6]:
```suggestion
- New integration providing generic enrichment commands.
- Includes enhancements to the readable output for the following commands:
   -  `cve`
   - `mati-get-actor`
   - `mati-get-malware`
   - `mati-get-campaign`
##### Mandiant Feed
New integration providing an indicator feed.
```

File: Packs/MandiantAdvantageThreatIntelligence/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+##### Mandiant Advantage Threat Intelligence
+- This pack now includes individual Feed and Enrichment integrations. Therefore this integration is now deprecated. Please follow the stpes in the documentation to update your integration with Mandiant.
```
Comment (ShirleyDenkberg) [inline Packs/MandiantAdvantageThreatIntelligence/ReleaseNotes/1_1_0.md:8]:
```suggestion
This pack now includes individual Feed and Enrichment integrations. Therefore this integration is now deprecated. Please follow the steps in the documentation to update your integration with Mandiant.
```

**Comment by ShirleyDenkberg** [comment]
@kgal-pan @barryyosi-panw @MLainer1 Doc review completed.

**Comment by thefrieddan1** [comment]
> @kgal-pan - FYI that circleci checks are now passing. Please let me know if anything else is required from my side.

Great. There are now comments from our technical writing expert.
Our next step will be to setup a demo or you can send me a recording of the new features you have added.

**Comment by adamlevymandiant** [comment]
Happy to set up a demo if you want to let me know a few times that work for you?

Should I send a new commit addressing the feedback from the tech writing expert? Will that trigger a whole set of new reviews if I do that?

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**